packages/client/src/index.ts

```diff
@@ -12,61 +11,60 @@ export const extractBearerToken = (authorization: string): string => {
     typeof authorization !== 'string' ||
```

Comment:
redundant type check?

Reply:
some developers may use our packages without typescripts, then will the type definition still work?

Reply:
if they are using JS, IDEs will still read type definitions for code intelligence and we'll also offer docs to instruct the developer. if they persist to ignore all these hints, they we can just let they go.

APIs should be guarded since they are public and we cannot trust any input. unlike SDKs, which are for "internal" use normally.

Reply:
agreed, type check will be removed, even if we need that in the future, an additional layer is better.